webapp/views/wpt-results.js

```diff
@@ -218,7 +230,7 @@ class WPTResults extends WPTColors(WPTFlags(PathInfo(LoadingState(TestRunsUIBase
 
                 <template is="dom-repeat" items="{{testRuns}}" as="testRun">
                   <template is="dom-if" if="[[ canAmendMetadata(node, index, testRun) ]]">
-                    <td class\$="numbers triage [[ testResultClass(node, index, testRun, 'passes') ]]" onclick="[[openAmendMetadata(index, node)]]">
+                    <td class\$="numbers triage [[ testResultClass(node, index, testRun, 'passes') ]]" data-index$="[[index]]" data-test$="[[node.path]]" on-click="handleSelectMetadata">
```

Comment:
`handleSelectMetadata` should probably be defined the same way as the previous `openAmendMetadata`: a function factory that takes `index` and `node` and returns a function, this avoids having to store lots of data attributes on the DOM.